docs/data/data-grid/components/toolbar/GridToolbarDefault.js

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { DataGrid, Grid, GridToolbarQuickFilter } from '@mui/x-data-grid';
+import { useDemoData } from '@mui/x-data-grid-generator';
+import Tooltip from '@mui/material/Tooltip';
+import Menu from '@mui/material/Menu';
+import Badge from '@mui/material/Badge';
+import ViewColumnIcon from '@mui/icons-material/ViewColumn';
+import FilterListIcon from '@mui/icons-material/FilterList';
+import PrintIcon from '@mui/icons-material/Print';
+import FileDownloadIcon from '@mui/icons-material/FileDownload';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import MenuItem from '@mui/material/MenuItem';
+
+function Toolbar() {
+  const [downloadMenuOpen, setDownloadMenuOpen] = React.useState(false);
+  const downloadMenuTriggerRef = React.useRef(null);
+
+  return (
+    <Grid.Toolbar.Root>
+      <Tooltip title="Columns">
+        <Grid.ColumnsPanel.Trigger render={<Grid.Toolbar.Button />}>
+          <ViewColumnIcon fontSize="small" />
+        </Grid.ColumnsPanel.Trigger>
+      </Tooltip>
+
+      <Tooltip title="Filters">
+        <Grid.FilterPanel.Trigger
```

Comment:
We need to set up some rules for deciding whether we should provide a wrapper component or not.
For example, `Tooltip` here is taken directly from Material UI, which makes sense to me. But how do we explain this?

Reply:
> We need to set up some rules for deciding whether we should provide a wrapper component or not.

"Are we actually using it internally in the grid?"